company-box-doc.el

```diff
@@ -77,7 +82,11 @@
           (box-width (frame-pixel-width (company-box--get-frame)))
           (window (frame-root-window frame))
           (frame-resize-pixelwise t)
-          ((width . height) (window-text-pixel-size window nil nil 10000 10000))
+          ((width . height)
+           (if company-box-doc-no-wrap
+               (window-text-pixel-size window nil nil 10000 10000)
+               (window-text-pixel-size window nil nil (- (frame-outer-width) (+ 20 (+ box-width (nth 0 box-position)))) (- (frame-outer-height) 20))
```

Comment:
What is this magic number 20?

Reply:
It's just a buffer to allow for the doc window's border; otherwise the text goes up to the very edge of the frame, clipping the border. I thought it looked better this way.

Reply:
Can you share a before and after screenshot?